### PR TITLE
[PWN-8446] First step

### PR DESCRIPTION
**Pull request by lisemyon**
## Link jira to issue
https://p2pvalidator.atlassian.net/browse/PWN-8446

## Description of the changes
-

### CODE REVIEW

File: p2p_wallet/Scenes/Main/BankTransfer/StrigaRegistration/FirstStep/StrigaRegistrationFirstStepCoordinator.swift [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+            switch result {
+            case .item(let item):
+                viewModel?.selectedCountryOfBirth = item as? Country
+            case .cancel: break
+            }
+        }.store(in: &subscriptions)
+
+        parent.pushViewController(vc, animated: true)
+
+        return result.first().eraseToAnyPublisher()
```
Comment (sidorov-panda) [inline p2p_wallet/Scenes/Main/BankTransfer/StrigaRegistration/FirstStep/StrigaRegistrationFirstStepCoordinator.swift:46]: If you have a chance - get rid of result subject to Publishers.Merge. It's easier to see all result cases in one place

Comment (lisemyon) [reply p2p_wallet/Scenes/Main/BankTransfer/StrigaRegistration/FirstStep/StrigaRegistrationFirstStepCoordinator.swift]: done it, pls check

**Review by bigearsenal — CHANGES_REQUESTED** [review]

**Comment by bigearsenal** [comment]
@lisemyon remove WIP also, as you have done this task

**Comment by lisemyon** [comment]
> @lisemyon remove WIP also, as you have done this task

I have not, I still need to do phone number formatting

**Pull request merged**